### DISCUSSION TITLE
Using MLForecast as a Global Forecasting Model wrapper

### DISCUSSION THREAD

**Discussion by giordanoalvarienel** [Nixtla/mlforecast#516] [Q&A]
Hi all!
I’m working on building a global forecasting model trained on several thousand unique_ids. These series are quite sparse and have different time spans, so some may only appear in the training set while others only in the test set. I’d like to implement this with Nixtla, but I’m not entirely sure if this setup is compatible with the library requirements.

From my initial trials, it seems that each unique_id needs to be continuous and complete. Is that correct? Or am I missing a way to handle sparse and incomplete series within Nixtla? I’d really appreciate any guidance, as I really like your libraries and would love to use them for this project.

Thanks!
